### DISCUSSION TITLE
[AtroPIM] Attribute Tabs vs. Attribute Groups

### DISCUSSION THREAD

**Discussion by marcus-at-localhost** [atrocore/atrocore#369] [Q&A] [answered]
What's the use cases for Attribute Tabs vs Attribute Groups? When do you use one or the other?

**Comment by atrolex** [comment, marked as answer]
Attribute Tabs can be shown or hidden (configurable in Administration / Layouts). They can be used for attributes needed by different departments, e.g. marketing/sales and support. Support department may need to see attributes which are not needed for marketing department (we plan to add the possibility to apply different layouts for some page, so it will be more clear than).

Attribute groups should be used to group attributes within certain attribute tab. Attribute group can also be used to mass link attributes of this group with a product family of a product.

Both attribute tabs and attribute groups are optional. So everybody can decide how it is better to use that in his/her particular case.
